.travis.yml

```diff
@@ -23,8 +23,10 @@ before_install:
   - docker ps -a
 
 install:
-  - go get -u github.com/pilosa/go-pilosa
-  - cd "$GOPATH/src/github.com/pilosa/go-pilosa" && git checkout v0.9.0 && cd "$TRAVIS_BUILD_DIR"
+  - go get -u github.com/golang/dep/cmd/dep
+  - cd "$TRAVIS_BUILD_DIR"
+  - touch Gopkg.toml
```

Comment:
we could just `dep init`

Reply:
I tried `dep init` but the problem is that it creates `Gopkg` files and locks the revision. After that any `dep ensure -add` or `dep ensure -update` doesn't work when I try to upgrade to the master (because lock is already set).
And for `pilosa/pilosa` we need to use master (waiting for 1.0.2).